x/bounty/types/program.go

```diff
@@ -1,7 +1,41 @@
 package types
 
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
 // Programs is an array of program
 type Programs []Program
 
 // Findings is an array of finding
 type Findings []Finding
+
+func (m *Program) Valid() bool {
```

Comment:
This should be named as something like `ValidateBasic` like in here https://github.com/cosmos/cosmos-sdk/blob/b75c29fc15d3320ec0c7596dbd7c787c48dccad8/x/gov/types/proposal.go#L206-L207